mozsci/map_train.py

```diff
@@ -89,7 +91,7 @@ def _run_kfold(self):
         if type(errors[0]['train']) == dict:
             for k in ['train', 'test']:
                 ret[k] = {}
-                for error_type in errors[0]['train'].keys():
+                for error_type in list(errors[0]['train'].keys()):
```

Comment:
You shouldn't need to explicitly invoke `list` here. That will materialize the keys into a `list`, but it is already an iterable, so the `for` loop will just work.